sokol_app.h

```diff
@@ -11362,7 +11483,7 @@ SOKOL_API_IMPL void sapp_set_clipboard_string(const char* str) {
     #elif defined(_SAPP_WIN32)
         _sapp_win32_set_clipboard_string(str);
     #else
-        /* not implemented */
+        _sapp_x11_set_clipboard_string(str);
```

Comment:
Here's why the Android and iOS builds also fail: the `_sapp_x11_set_clipboard_string` needs to go into its own `#elif defined(_SAPP_LINUX)` block, and the `#else` block needs to remain empty.